app/src/main/java/com/duckduckgo/widget/SearchWidget.kt

```diff
@@ -24,10 +24,10 @@ import android.widget.RemoteViews
 import com.duckduckgo.app.browser.BrowserActivity
 import com.duckduckgo.app.browser.R
 
-/**
- * Implementation of App Widget functionality.
- */
-class SearchWidget : AppWidgetProvider() {
+
+class SearchWidgetLight : SearchWidget(R.layout.search_widget_light)
```

Comment:
I like this approach; should we do it for both light and dark?

    class SearchWidgetLight : SearchWidget(R.layout.search_widget_light)
    class SearchWidgetDark : SearchWidget(R.layout.search_widget)

And maybe we should rename `R.layout.search_widget` to have `_dark` on the end for better clarity/symmetry?

Reply:
I would prefer that too (and initially had that exact setup) except if we change the class name we break all the existing widgets that name.

So I've gone with concept of "dark" as the "default" and kept the resources the same to keep things consistent.